README.md

```diff
@@ -2,14 +2,21 @@
 
 # Groundwork API Client
 
-API client for [GroundWork Renewables](grndwork.com)
+API client for [GroundWork Renewables](https://grndwork.com)
+
 
 ## Installation
+JavaScript:
 
 ```
 $ npm install @grndwork/api-client
 ```
 
+Python:
+```
+pip install grndwork_python_client
+```
```

Comment:
grndwork_api_client

Reply:
Changed to grndwork_api_client since [hyphens are not allowed in identifiers](https://bugs.python.org/issue909226).

Reply:
Changed to grndwork_api_client for Python only per discussion w/ Ian.